app/javascript/src/component_activated_question_menu.js

```diff
@@ -31,7 +30,10 @@ class QuestionMenu extends ActivatedMenu {
       onSetRequired: function(){} // Called at end of set required function
     }, config));
 
-    $node.on("menuselect", QuestionMenu.selection.bind(this) );
+    //$node.on("menuselect", QuestionMenu.selection.bind(this) );
```

Comment:
can we take out the comment? unless we're coming back to this soon?

Reply:
yeah, sorry - that can definitely go.